src/plugins/vis_type_timeseries/common/agg_lookup.ts

```diff
@@ -32,7 +32,7 @@ export const lookup: Record<string, string> = {
     defaultMessage: 'Percentile',
   }),
   percentile_rank: i18n.translate('visTypeTimeseries.aggLookup.percentileRankLabel', {
-    defaultMessage: 'Percentile Rank',
+    defaultMessage: 'Percentile rank',
```

Comment:
I wonder if only changing the uppercase to lowercase influences japanese and chinese? 🤔 Probably not, but if it is, we should remove the copy for `visTypeTimeseries.aggLookup.percentileRankLabel` from the translation files :)

Reply:
Marta except from the fact that you are right, I was thinking about it and I decided to not change it because I see that the rest of aggs are using this format i.e.`Value Count`